public/app/plugins/panel/nodeGraph/utils.ts

```diff
@@ -138,12 +143,36 @@ export function processNodes(
     return { nodes: [], edges: [] };
   }
 
+  let hasFixed: boolean | undefined = undefined;
```

Comment:
For a bit more clarity can we rename this to `hasFixedPositions` or something like that?